style.css

```diff
@@ -1006,3 +1093,1378 @@
   border-radius: 10px;
   align-items: center;
 }
+
+@media screen and (min-width: 768px) {
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  :root {
+    --font1: 'Poppins';
+    --font2: 'Roboto', sans-serif;
+    --font3: 'Crete Round', sans-serif;
+    --font4: 'Inter', sans-serif;
+    --font5: 'InaiMathi', sans-serif;
+  }
+
+  .header nav {
+    font-family: var(--font2);
+    color: #fff;
+    font-style: normal;
+    font-weight: 100;
+    font-size: 18px;
+    line-height: 24px;
+    width: 100%;
+    height: 72px;
+    background: #3c3a39;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 12px 24px;
+    text-align: center;
+  }
+
+  .header nav .name {
+    text-decoration: none;
+    color: #fff;
+  }
+
+  .header nav .menu-icon {
+    color: #fff;
+    width: 40px;
+    height: 40px;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    display: none;
+  }
+
+  .website {
+    /* display: block; */
+    width: 300px;
+    text-decoration: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    color: #fff;
+  }
+
+  .website .portfolio {
+    width: 53px;
+    height: 20px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    text-decoration: none;
+    margin-top: 26px;
+    margin-left: 10px;
+    margin-right: 15px;
+    margin-bottom: 26px;
+    padding: 12px 24px;
+  }
+
+  .About-page {
+    width: 45px;
+    height: 20px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    text-decoration: none;
+    color: #fff;
+  }
+
+  .contact {
+    width: 59px;
+    height: 20px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .container-1 {
+    width: 100%;
+    min-height: 3455px;
+  }
+
+  .container-1 .section {
+    padding: 120px 24px 120px 24px;
+    width: 100%;
+    height: 590px;
+    background: #1c1a19;
+    background-image: url(images/background_img_desktop.png);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+  }
```

Comment:
- [ ] It looks like the background images you described here have not been applied. look at the screenshot of what I have

 **Your design** 
![image](https://user-images.githubusercontent.com/90222110/188198000-f0561a44-7b50-4f11-a14b-1a5616bb14ec.png)

This may be because the image you referenced does not exist please check it again and make sure it is being shown as it is among project requirements to stick to the Figma design given 😃